### PR TITLE
Build plugin with sbt 1.5.5 with minimum sbt version unchanged.

### PR DESCRIPTION
**Pull request by myyk**
I don't think this causes any harm, but I haven't done sbt plugin development in a while.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,7 +14,11 @@ lazy val `sbt-scommons-plugin` = (project in file("."))
     organization := "org.scommons.sbt",
     name := "sbt-scommons-plugin",
     description := "Sbt auto-plugin with common tasks/utils for Scala Commons modules",
-    scalaVersion := "2.12.7",
```
Comment (myyk) [inline build.sbt:17]: I may be wrong, but I think this value doesn't need to be specified.

**Comment by myyk** [comment]
Hrm... I'm not sure I understand why the build is failing in Travis. Locally, I'm seeing different failures trying to crossbuild 1.0.2 of master even.

**Comment by myyk** [comment]
Oh maybe this is the problem. When I change `scalaBinaryVersion` down to `1.0.2` which would enforce that the plugin would build compatible for 1.0.2. Then the `x / y` syntax is not compatible for both.

I think this could be maintained by forking the 6 or so files that have this incompatibility or just choose to bump the minimum version SBT.

**Comment by viktor-podzigun** [comment]
Addressed as part of #11

**Pull request closed without merging**